squeal-postgresql/src/Squeal/PostgreSQL/Definition/Constraint.hs

```diff
@@ -286,16 +289,17 @@ foreignKey keys parent refs ondel onupd = UnsafeTableConstraintExpression $
 
 -- | A constraint synonym between types involved in a foreign key constraint.
 type ForeignKeyed db
-  sch
-  schema
+  sch0 sch1
+  schema0 schema1
```

Comment:
this was my mistake but the variable order goes parent-child here and then child-parent below

Reply:
Yeah, my bad for blindly copying it over. Will make sure I understand it next time I push.